docs/data/migration/migration-tree-view-v6/migration-tree-view-v6.md

```diff
@@ -25,3 +25,40 @@ In `package.json`, change the version of the tree view package to `next`.
 
 Since `v7` is a major release, it contains changes that affect the public API.
 These changes were done for consistency, improved stability and to make room for new features.
+
+### Rename `onNodeToggle`, `expanded` and `defaultExpanded`
+
+The expansion props have been renamed to better describe their behaviors:
+
+| Old name          | New name                |
+| :---------------- | :---------------------- |
+| `onNodeToggle`    | `onExpandedNodesChange` |
+| `expanded`        | `expandedNodes`         |
+| `defaultExpanded` | `defaultExpandedNodes`  |
+
+```diff
+  <TreeView
+-   onNodeToggle={handleExpansionChange}
++   onExpandedNodesChange={handleExpansionChange}
+
+-   expanded={expandedNodes}
++   expandedNodes={expandedNodes}
+
+-   defaultExpanded={defaultExpandedNodes}
++   defaultExpandedNodes={defaultExpandedNodes}
+  />
+```
+
+:::info
+If you were using the `onNodeToggle` prop to react to the expansion or collapse of a specific node,
```

Comment:
```suggestion
If you were using the `onNodeToggle` prop to react on the expansion or collapse of a specific node,
```

Reply:
https://dictionary.cambridge.org/dictionary/english/react are you sure for this one?

Reply:
looking at the dictionary it seems wrong. Dunno ... it felt more right, but i might be mistaken. 🤷🏼 You can ignore that!